second-edition/src/ch06-02-match.md

```diff
@@ -118,18 +117,18 @@ enum Coin {
 }
 ```
 
-<span class="caption">Listing 6-4: A `Coin` enum where the `Quarter` variant
-also holds a `UsState` value</span>
+<span class="caption">Listado 6-4: Una enumeración `Coin` donde la variante `Quarter` 
+tambien contiene un valor `UsState`</span>
 
-Let’s imagine that a friend of ours is trying to collect all 50 state quarters.
-While we sort our loose change by coin type, we’ll also call out the name of
-the state associated with each quarter so if it’s one our friend doesn’t have,
-they can add it to their collection.
+Imaginemos que un amigo nuestro esta intentando recolectar los 50 alojamientos del estado.
+Mientras clasificamos nuestro cambio impreciso por tipo de moneda, tambien llamamos el
+nombre del estado asociado con cada alojamiento asique si uno de nuestros amigos no tiene,
+ellos pueden agregarlo a su colección.
 
-In the match expression for this code, we add a variable called `state` to the
-pattern that matches values of the variant `Coin::Quarter`. When a
-`Coin::Quarter` matches, the `state` variable will bind to the value of that
-quarter’s state. Then we can use `state` in the code for that arm, like so:
+En la expresión de coincidencia para este código, agregamos una variable llamada `state` al
+patrón que coincide con el valor de la variante `Coin::Quarter`. Cuando un
+`Coin::Quarter` coincide, la variable `state` se juntará al valor del 
+estado de alojamientos. Entonces podemos usar `state` en el código para ese brazo, asi:
 
 ```rust
 # #[derive(Debug)]
```

Comment:
Again, please try and translate the variables used here, included UsState (maybe EstadosUs)

Reply:
fixed, look at it now, to see if it's okay